styles.css

```diff
@@ -240,16 +247,6 @@ form {
     display: flex;
 }
 
-/* toggling this class removes the enter button*/
-/*.enter{
-    display: none;
-}
```

Comment:
I would suggest removing the classes from the HTML document as they have become obsolete in this story; I would say that the same likely applies for the classes Input and Outcome that were used to present the changeability of the html element.

Reply:
Removed :)